src/components/DateTime/DateTime.jsx

```diff
@@ -39,3 +43,25 @@ export function DateTime(props) {
 		</span>
 	);
 }
+
+function validateDateTime(value) {
+	if (value == null) return 'Invalid Date';
+
+	// Reject non-numeric values
+	if ((typeof value !== 'number') || !Number.isFinite(value)) return new Date(value);
```

Comment:
@aringocode why this? `!Number.isFinite(value)`  - is it because of bigInt?

Reply:
Reject infinite, NaN, or negative numbers